util/predicates/configmap_predicates.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package predicates
+
+import (
+	"github.com/go-logr/logr"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+)
+
+// ConfigMapCreate returns a predicate that returns true for a ConfigMap create event
+func ConfigMapCreateUpdate(logger logr.Logger) predicate.Funcs {
```

Comment:
nit: This doesn't actually have any logic specific to ConfigMaps, so I think more generic naming makes it clear that it can be used for allowing updates only on create.
```suggestion
func ResourceCreate(logger logr.Logger) predicate.Funcs {
```